airflow_valohai_plugin/operators/valohai_submit_execution_operator.py

```diff
@@ -108,8 +108,8 @@ def execute(self, context):
                     'name': output['name'],
                     'ctime': output['ctime'],
                     'size': output['size'],
-                    'uri': output['uri'],
-                    'id': output['id']
+                    'id': output['id'],
+                    'url': hook.get_output_url(output['id']),
```

Comment:
Let's not get rid of `uri` there 🙏

Also, I'm wondering if we should handle this lazily and do the actual API call to get the output URL in the download operator instead. That way, if download operator isn't called, we won't have to make this API call.